open-craft/demo-mfe
# [STAR-2442] Use providence to handle all state management.

Refactors state management to use Providence

**JIRA tickets**: https://tasks.opencraft.com/browse/STAR-2442

**Testing instructions**:

1. Have a working and running Devstack
2. [Install NVM](https://github.com/nvm-sh/nvm), then, in the repository root:
3. `nvm install`
4. `nvm use`
5. `npm install -g npm@8.6.0`
6. `npm install`
7. `npm start`
8. Visit http://localhost:8080/catalog/ and verify the catalog displays.
